src/component/component.yaml

```diff
@@ -0,0 +1,28 @@
+id: PCLCrypto
+name: Cryptography Plugin
+publisher: Andrew Arnott
+publisher-url: https://twitter.com/aarnott
+summary: PCL Storage provides a consistent, portable set of crypto APIs across .NET platforms.
```

Comment:
"PCL Storage", huh? I'm guessing you copied and pasted this from elsewhere in PCLCrypto... somewhere that I need to fix due to a very old copy-paste bug. :(